client/src/pages/RoomPage.tsx

```diff
@@ -177,22 +177,11 @@ function RoomPage({ name }: { name: string }): JSX.Element {
           title="Press m to mute/unmute"
           w="4%"
           bg="rgb(0 0 0 / 60%)"
-          onClick={function (){
-            updateMute(!mute);
-          }}
+          onClick={updateMute(!mute)}
         >
           {mute ? <MicOffIcon /> : <MicIcon />}
         </Button>
-        <Button
-          pos="absolute"
-          bottom="1rem"
-          left="35%"
-          w="30%"
-          onClick={function (){
-            history.go(0);
-          }}
-          bg="red"
-        >
+        <Button pos="absolute" bottom="1rem" left="35%" w="30%" onClick={history.go(0)} bg="red">
```

Comment:
With this syntax, history.go(0) gets called whenever its mounted
=> introduces infininte loop